pandas/core/indexing.py

```diff
@@ -539,14 +539,25 @@ def at(self) -> _AtIndexer:
         Raises
         ------
         KeyError
-            If 'label' does not exist in DataFrame.
+            * If getting a value and 'label' does not exist in a DataFrame or
```

Comment:
Mind Adding a `Notes` section linking to the user guide that was referenced in the original issue?

https://pandas.pydata.org/docs/dev/user_guide/indexing.html#fast-scalar-value-getting-and-setting

Reply:
Done